src/main/java/seedu/address/model/person/ClassId.java

```diff
@@ -33,7 +33,7 @@ public ClassId(String classId) {
      * Returns true if a given string is a valid grade.
      */
     public static boolean isValidClassId(String test) {
```

Comment:
Should we consider renaming this variable? I think this is what the original AB3 calls it, but `test` seems a bit vague to me. Maybe something like `test_classId` would be better?

Reply:
sounds good! let me fix that